### ISSUE TITLE
Release v1.2.1: Env var name collision with dbt-core

### ISSUE THREAD

**Issue by will-misslin-ironclad** [mwhitaker/dbt-action#39]
With the latest release, actions have begun to fail due to misconfiguration of logging.

```log
dbt project folder set as: "/github/workspace/data_marts"
trying to parse bigquery token
success parsing plain token
saving console output in "/github/workspace/data_marts/dbt.log"
Error: 10:23:[20](https://github.com/**********/actions/runs/**********/jobs/**********#step:6:21).255872 [error] [MainThread]: Encountered an error:
[Errno 20] Not a directory: '/github/workspace/data_marts/dbt.log/dbt.log'
10:23:20  Encountered an error:
[Errno 20] Not a directory: '/github/workspace/data_marts/dbt.log/dbt.log'
Error: 10:[23](https://github.com/**********/actions/runs/**********/jobs/**********#step:6:24):20.[25](https://github.com/**********/actions/runs/**********/jobs/**********#step:6:26)7841 [error] [MainThread]: Traceback (most recent call last):
```

It seems that dbt-core expects that the [new `DBT_LOG_PATH` env var (also available as the log-path yml config) points to a directory](https://docs.getdbt.com/reference/project-configs/log-path). However, [the `DBT_LOG_PATH` in dbt-action seems to serve different purpose at points at the log file itself.](https://github.com/mwhitaker/dbt-action/blob/9f09e92cb0b8f377747603313d9bd8c2b2b5c01a/entrypoint.sh#L36).

Perhaps the `DBT_LOG_PATH` env var in dbt-action is no longer necessary given the new dbt-core log-path config option? Happy to submit a PR if you are accepting them right now! 🙂

**Comment by joelazar** [comment]
I've opened a pr for it --> https://github.com/mwhitaker/dbt-action/pull/40

**Comment by mwhitaker** [comment]
Thank you both! When I created this action I wanted to capture the actual screen output, hence the `DBT_LOG_PATH` environment variable. Maybe it's no longer needed.

@joelazar I am still getting around to testing this myself, but did you test your PR that just renaming for now does the trick?

**Comment by joelazar** [comment]
> Thank you both! When I created this action I wanted to capture the actual screen output, hence the `DBT_LOG_PATH` environment variable. Maybe it's no longer needed.
> 
> @joelazar I am still getting around to testing this myself, but did you test your PR that just renaming for now does the trick?

@mwhitaker, yes, and it did the trick :)

**Comment by mwhitaker** [comment]
Thank you! Merged #40
